plugins/catalog/src/alpha.tsx

```diff
@@ -65,12 +81,191 @@ export const CatalogSearchResultListItemExtension =
+    return <EntityTagPicker />;
+  },
+});
+
+const CatalogEntityKindFilter = createCatalogFilterExtension({
+  id: 'entity.kind',
+  configSchema: createSchemaFromZod(z =>
+    z.object({
+      initialFilter: z.string().default('component'),
+    }),
+  ),
+  loader: async ({ config }) => {
+    const { EntityKindPicker } = await import(
+      '@backstage/plugin-catalog-react'
+    );
+    return <EntityKindPicker initialFilter={config.initialFilter} />;
+  },
+});
+
+const CatalogEntityTypeFilter = createCatalogFilterExtension({
+  id: 'entity.type',
+  loader: async () => {
+    const { EntityTypePicker } = await import(
+      '@backstage/plugin-catalog-react'
+    );
+    return <EntityTypePicker />;
+  },
+});
+
+const CatalogEntityOwnerFilter = createCatalogFilterExtension({
+  id: 'entity.mode',
+  configSchema: createSchemaFromZod(z =>
+    z.object({
+      mode: z.enum(['owners-only', 'all']).optional(),
+    }),
+  ),
+  loader: async ({ config }) => {
+    const { EntityOwnerPicker } = await import(
+      '@backstage/plugin-catalog-react'
+    );
+    return <EntityOwnerPicker mode={config.mode} />;
+  },
+});
+
+const CatalogEntityNamespaceFilter = createCatalogFilterExtension({
+  id: 'entity.namespace',
+  loader: async () => {
+    const { EntityNamespacePicker } = await import(
+      '@backstage/plugin-catalog-react'
+    );
+    return <EntityNamespacePicker />;
+  },
+});
+
+const CatalogEntityLifecycleFilter = createCatalogFilterExtension({
+  id: 'entity.lifecycle',
+  loader: async () => {
+    const { EntityLifecyclePicker } = await import(
+      '@backstage/plugin-catalog-react'
+    );
+    return <EntityLifecyclePicker />;
+  },
+});
+
+const CatalogEntityProcessingStatusFilter = createCatalogFilterExtension({
+  id: 'entity.processing.status',
+  loader: async () => {
+    const { EntityProcessingStatusPicker } = await import(
+      '@backstage/plugin-catalog-react'
+    );
+    return <EntityProcessingStatusPicker />;
+  },
+});
+
+const CatalogUserListFilter = createCatalogFilterExtension({
+  id: 'user.list',
+  configSchema: createSchemaFromZod(z =>
+    z.object({
+      initialFilter: z.enum(['owned', 'starred', 'all']).default('owned'),
+    }),
+  ),
+  loader: async ({ config }) => {
+    const { UserListPicker } = await import('@backstage/plugin-catalog-react');
+    return <UserListPicker initialFilter={config.initialFilter} />;
+  },
+});
+
+const CatalogIndexPage = createPageExtension({
+  id: 'catalog',
+  defaultPath: '/catalog',
+  routeRef: rootRouteRef,
+  inputs: {
+    filters: createExtensionInput({
+      element: coreExtensionData.reactElement,
+    }),
+  },
+  loader: async ({ inputs }) => {
+    const { BaseCatalogPage } = await import('./components/CatalogPage');
+    const filters = inputs.filters.map(filter => filter.element);
+    return <BaseCatalogPage filters={<>{filters}</>} />;
+  },
+});
+
+const CatalogEntityPage = createPageExtension({
+  id: 'catalog:entity',
```

Comment:
see above

Reply:
For this other catalog page same situation, I had to use the same id as the routeRef id...

Reply:
I'll update this once this pull request gets merged 🙌🏻 : https://github.com/backstage/backstage/pull/20574